priority-queue/Readme.md

```diff
@@ -4,43 +4,87 @@ This document describes the Priority Queue Pattern example from the guide [Cloud
 
 ## Running the Example
 
-You can run this example locally in the Visual Studio Windows Azure emulator. You can also run this example by deploying it to a Windows Azure Cloud Service.
+You can either run this example locally in the Visual Studio Windows Azure emulator or you can run it by deploying it to Function App Services.
 
-* Start Visual Studio using an account that has Administrator privileges ("Run as Administrator").
+* Provision an Azure Service Bus Namespace.
+* Start Visual Studio.
 * Open the solution you want to explore from the subfolders where you downloaded the examples.
-* Right-click on each role in Solution Explorer, select Properties, and ensure that the role is configured to generate diagnostic information.
-* Provision a Windows Azure Service Bus Namespace
-* Edit the ServiceBus.ConnectionString setting by changing "[SERVICE BUS CONNECTION STRING]" to your own Service Bus endpoint in the following files for every role in the solution:
-	* ServiceConfiguration.Cloud.csfg if you will run the example on Windows Azure.
-	* ServiceConfiguration.Local.csfg if you will run the example in the Windows Azure Emulator. (you can update settings by right-clicking on each of the roles and selecting Properties, or by editing the files directly in the main Visual Studio window.)
-* If you want to run the example in the local Windows Azure emulator:
-	* Press F5 in Visual Studio to start the example running.
-	* Open the Windows Azure Compute Emulator UI from the icon in the notification area.
-	* Select each role in turn and view the diagnostic information generated by Trace statements in the code.
-* If you want to run the example on Windows Azure:
-	* Provision a Windows Azure Cloud Service and deploy the application to it from Visual Studio.
-	* Open the Server Explorer Window in Visual Studio and expand the Windows Azure entry.
-	* Expand Cloud Services and then expand the entry for the solution you deployed.
-	* Right-click each role instance and select View Diagnostics Data to see the diagnostic information generated by Trace statements in the code. This is written to the WADLogsTable in the Storage/Development/Tables section.
+* Edit the local.settings.json file in all the projects and change the ServiceBusConnection__fullyQualifiedNamespace setting by changing the placeholder "<service_bus_namespace>"  to your Azure Service Bus Namespace name.
 
+* If you want to run the example in the local Windows Azure emulator:
+	* Set the PriorityQueueSender project as startup.
+	* Press F5 in Visual Studio to start the example running. The function will start sending messages to the topic, every 30 seconds.
```

Comment:
What topic?  You only asked the user to create a namespace.

Likewise, where are the instructions to set up the topic filtering based on the custom property you added to the message?

Reply:
Looks good, with the exception of one thing:

If you're going to use `highPriority` and `lowPriority` here in the instructions, you'll then need to update the bindings on your two consumer functions. Otherwise, you can update the values here to match the bindings.

Reply:
Bindings updated. Thanks for the review !

Reply:
you got one of the two.  Comment above updated to reflect the partial change.

Reply:
Got the second one now. Thanks.